packages/user-group-service/src/groups/schema.ts

```diff
@@ -2,7 +2,7 @@ import { Schema, Document, Model, model } from 'mongoose';
 
 export const GroupSchema: Schema = new Schema( {
   name: { type: String, unique: true, required: true, },
-  ldapCommonName: { type: String, unique: true, required: true, },
+  cn: { type: String, unique: true, required: true, },
```

Comment:
**Just a note:**
This will require a change in the User Group SPA, since it currently uses the `ldapCommonName` field.